phpBB/phpbb/db/migration/data/v33x/add_bbcode_font_icon.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+
+namespace phpbb\db\migration\data\v33x;
+
+class add_bbcode_font_icon extends \phpbb\db\migration\migration
+{
+	public function effectively_installed()
+	{
+		return $this->db_tools->sql_column_exists($this->table_prefix . 'bbcodes', 'bbcode_font_icon');
+	}
+
+	static public function depends_on()
+	{
+		return [
+			'\phpbb\db\migration\data\v330\v330'
+		];
+	}
+
+	public function update_schema()
+	{
+		return [
+			'add_columns'	=> [
+				$this->table_prefix . 'bbcodes'	=> [
+					'bbcode_font_icon'	=> ['VCHAR_UNI:255', ''],
```

Comment:
Most font icon names are small. And none have unicode text to them. You could probably use `VCHAR:50` instead (and limit strlen to 50 in the ACP code)